app.js

```diff
@@ -1,11 +1,44 @@
 var express = require('express');
+var bodyParser = require('body-parser');
 var app = express();
 
 var port = process.env.PORT || 4000;
 
-app.get('/', function(req, res) {
-    //res.send('<html><head></head><body><h1>Teams:</h1></body></html>');
-    res.send(['acs-cognitive-team', 'acs-fabric-team', 'acs-appdev-team', 'acs-database-team', 'acs-database-db2', 'acs-compute-team', 'acs-security-team', 'acs-network-team', 'acs-storage-team', 'acs-virt-team', 'acsvpc']);
-});
+let domains = [
+    {
+      "team": 1,
+      "acs-domain": "acs-cognitive",
+      "focus": "service issue and data loss related queries"
+    },
+    {
+      "team": 2,
+      "acs-domain": "acs-fabric",
+      "focus": "service functions"
+    },
+    {
+      "team": 3,
+      "acs-domain": "acs-security",
+      "focus": "service security"
+    }
+  ];
 
-app.listen(port, () => console.log(['acs-cognitive-team,acs-fabric-team,acs-appdev-team, acs-database-team, acs-database-db2, acs-compute-team, acs-security-team,acs-network-team, acs-storage-team, acs-virt-team, acsvpc, goat-team']));
\ No newline at end of file
+  //Example request: http://localhost:4000/domains
+  app.get('/domains', function(req, res) {
+    res.send(domains);
+  });
+
+var teamMappings = [
```

Comment:
@saidah-neves careful here how you define `teamMappings`. Since you are using both square brackets `[` and  curly brackets`{` you are essentially defining an array of one object. Therefore, when you use `arr.find` here: https://github.com/saidah-neves/LiveEngage-Technical-Domains/pull/9/files#diff-e07d531ac040ce3f40e0ce632ac2a059d7cd60f20e61f78268ac3be015b3b28fR40 You are just examining this one object.

I would change this line to read
```
var teamMappings = {
  "Assistant": "acsCognitive",
  "IAM": "acsCognitive"
  ...
}
```
Then on line 40, I would set team name to be

```
var teamName = teampMappings[keyword];
```

Reply:
This way you're accessing a property on the `teamName` object